core_fw/ipc_radio/Kconfig

```
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+mainmenu "Nordic ipc_radio sample"
+
+config IPC_RADIO_802154
+	bool "IPC IEEE 802.15.4 radio"
+	select NRF_802154_SER_RADIO
+	default n
+	help
+	  Enable the IPC IEEE 802.15.4 radio serialization.
+
+config IPC_RADIO_BT
+	bool "IPC Bluetooth"
+	default n
+	help
+	  Enable the IPC Bluetooth radio serialization.
+
+if IPC_RADIO_BT
+
+choice IPC_RADIO_BT_SER
+	default IPC_RADIO_BT_HCI_IPC
+	prompt "Bluetooth serialization type"
+	help
+	  Type of the IPC Bluetooth radio serialization used.
+
+config IPC_RADIO_BT_HCI_IPC
+	bool "Bluetooth serialization over IPC Service"
```

I think the important information here is which Bluetooth layer is serialized, not which service is used
```suggestion
	bool "Bluetooth HCI serialization"
```